### PR TITLE
Update Pipeline Configuration with missing confs

### PR DESCRIPTION
**Pull request by tommyblue**
#### :question: What

we missed a bunch of conf sections

### CODE REVIEW

**Comment by arl** [comment]
Not sure about documenting the `[csv]` section since ideally it should go as it's specific to a particular Record implementation, which might not even be present if baker is used as a library.
The rest LGTM

**Comment by tommyblue** [comment]
> Not sure about documenting the `[csv]` section since ideally it should go as it's specific to a particular Record implementation, which might not even be present if baker is used as a library.

I see, but it's also true that the docs should document what's in the project. I think we could add it (as it's present now) and we'll remove once removed from the code

**Comment by arl** [comment]
> > Not sure about documenting the `[csv]` section since ideally it should go as it's specific to a particular Record implementation, which might not even be present if baker is used as a library.
> 
> I see, but it's also true that the docs should document what's in the project. I think we could add it (as it's present now) and we'll remove once removed from the code

The thing is that once you're documenting something, it's in theory harder to remove since you influence people to use it.

**Comment by tommyblue** [comment]
> The thing is that once you're documenting something, it's in theory harder to remove since you influence people to use it.

hehe, we already have a PR on that :smile: 
So let's choose quickly what to do then

File: content/en/docs/How Tos/pipeline_configuration.md [hunk 1]
```diff
@@ -83,6 +103,10 @@ can be specified, and in this case we simply provide the files option to List.
 Notice that List would accept http:// or even s3:// URLs there in addition to local paths,  
 and some more (run ./Baker-bin -help List in the help example for more details).
 
+`[filterchain]` defines the configuration for the whole filter chain. Filter-specific configuration
+are provided by `[[filter]]` (see below). The only accepted configuration is `procs = <int>` that
```
Comment (arl) [inline content/en/docs/How Tos/pipeline_configuration.md:107]: Either:
 - Filter-specific configuration is provided
 - Filter-specific configurations are provided

I guess

**Review by arl — APPROVED** [review]
Thanks
LGTM

There's just a grammar comment

**Pull request merged**